src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -1001,6 +1020,11 @@ List<Exception> generalResolutionExceptions
                                 Log.LogMessage(ChooseReferenceLoggingImportance(conflictCandidate), StringBuilderCache.GetStringAndRelease(logConflict));
                                 Log.LogMessage(MessageImportance.Low, StringBuilderCache.GetStringAndRelease(logDependencies));
                             }
+
+                            if (OutputUnresolvedAssemblyConflicts)
+                            {
+                                _unresolvedConflicts.Add(new TaskItem(assemblyName.Name));
```

Comment:
I would include the log message with the human-readable details in the metadata for this item.

We don't currently have a use case for programmatically processing data such as conflicting version numbers, how the assembly was referenced, etc.  I would say if it's straightforward to include, then go ahead and do so.  But if not, then skip it for now.

Reply:
Conflicting version numbers is easy, so I included that. How it's referenced is doable, but it's a little verbose, since there could be multiple levels of dependencies (A depends on B depends on C depends on D depends on the conflicting dll). I can add it later if we decide we need it, though.